code/style.css

```diff
@@ -1,4 +1,346 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+/*** General design ***/
+  body {
+    background: #89507f;
+  }
+
+/*** Media for mobile ***/
+  @media (min-width: 1px) and (max-width: 667px){
+  
+  /*** Header ***/
+    header {
+      background: white;
+      color: #89507f;
+      font-family: Arial, Helvetica, sans-serif;
+      display: flex;
+      justify-content: flex-start;
+      align-items: center;
+      height: 120px;
+      border-radius: 10px;
+      margin-bottom: 10px;
+    }
+
+    header img {
+      height: 100px;
+    }
+
+    h1 {
+      color: #89507f;
+      margin: 10px;
+    }
+
+    header a {
+     display: none;
+    }
+
+  /*** All images ***/
+    img {
+      border-radius: 10%;
+      }
+
+  /*** Section of news/cards ***/
+    .big-news {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start;
+      align-items: flex-start;
+      margin: 15px;
+      border-bottom-style: solid;
+      border-color: white;
+    }
+
+    .big-news img {
+      width: 300px;
+      height: auto;
+      margin: 5px;
+    }
+
+    .big-news-text {
+      padding: 15px;
+      color: white;
+      font-family: monospace;
+    }
+
+    .other-news-cards {
+      display: flex;
+      flex-direction: column;
+      margin-bottom: 10px;
+    }
+
+    .card {
+      margin: auto;
+      padding: 15px;
+      background: white;
+      box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+      width: 80%;
+      text-align: center;
+      color: #89507f;
+      font-family: monospace;
+      margin-bottom: 10px;
+      margin-top: 20px;
+      border-radius: 10px;
+    }
+
+    .card img {
+      max-width: 100%;
+      margin-left: auto;
+      margin-right: auto;
+      padding-top: 10px;
+      padding-bottom: 10px;
+      object-fit: cover;
+    }
+
+    .card:hover {
+      box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+    }
```

Comment:
I could almost not see this box-shadow though. Consider adding some more or a different color?